src/FlightDisplay/FlyViewWidgetLayer.qml

```diff
@@ -149,6 +149,15 @@ Item {
         property bool _verticalCenter: !QGroundControl.settingsManager.flyViewSettings.alternateInstrumentPanel.rawValue && !_showWinchControlMenu
     }
     
+    BatteryWidget {
+        id:                     batteryWidget
+        anchors.margins:        _toolsMargin
+        anchors.right:          parent.right
+        anchors.bottom:         winchControl.visible ? winchControl.top : parrent.bottom
+        width:                  _rightPanelWidth
+        availableHeight:        parent.height - (winchControl.height + instrumentPanel.height + ScreenTools.defaultFontPixelHeight * 2)
+    }
```

Comment:
Have you tested on different screen sizes / UI zooms?

Reply:
Yes I have tested it with upto 150% zoom and on a couple of different screens